src/pages/Home/components/Banner/styles.module.scss

```
@@ -0,0 +1,22 @@
+.homeMainImage {
```

имена классов пишем кебаб-кейсом, а уже в компонентах можно для обращения к классам использовать кэмэл-кейс
```suggestion
.home-main-image {
```

---

Почему-то в компоненте к классам стилей нельзя обращаться в кэмелкейсе если в файле со стилями классы прописаны через дефис. Приходится в компонентах писать styles['home-main-image'];